docs/pages/machine-id/introduction.mdx

```diff
@@ -58,13 +40,12 @@ Bots share a number of similarities with human users:
 
 ### Join tokens
 
-The main difference is that a bot does not "log in" using a static username and
-password. Instead, a join token is used. A join token is configured within
-Teleport and specifies which bot user it grants access to and what sort of
-proof (known as the join method) is needed to use this join token. This proof is
-typically an identity issued to the machine by the platform it runs on (e.g. AWS
-IAM). This process allows the machine to authenticate with the Teleport
-cluster without using a long-lived secret.
+A bot authenticates to Teleport with a **join token**, which prevents it from
```

Comment:
A join token can itself be a long-lived secret though. I kind of liked the "a bot does not log in" terminology.